proxycore/endpoint.go

```diff
@@ -98,7 +103,7 @@ func (r *defaultEndpointResolver) Resolve() ([]Endpoint, error) {
 		}
 		for _, addr := range addrs {
 			endpoints = append(endpoints, &defaultEndpoint{
-				net.JoinHostPort(addr, port),
+				addr: net.JoinHostPort(addr, port),
```

Comment:
Is there no need to add the tls config in this resolver?

Reply:
Not yet. TLS is not yet currently supported for proxy <--> Cassandra. More work would have to be done to support that.

Reply:
Oh, right. But it already does support TLS proxy <-> Astra right? Do you see a use case for cql proxy to be used with clusters that are not Astra?

Reply:
> But it already does support TLS proxy <-> Astra right?

yes

> Do you see a use case for cql proxy to be used with clusters that are not Astra?

It's mostly used for testing, and by at least one user. I have a dream of cql-proxy + lightweight drivers being used with C*. TLS + token-aware (I have this mostly done) are the few remaining things to make that possible. Also, it could be used as ingress for C* deployed on k8s. None of this is a priority currently. :)

Reply:
Ohhh that makes a lot of sense, that's super exciting 💯